README.md

```diff
@@ -215,13 +216,22 @@ Here's a list of supported alternative widgets for different JSONSchema data typ
   * `password`: an `input[type=password]` element is used;
   * by default, a regular `input[type=text]` element is used.
 
-The built-in string field also supports the JSONSchema `format` property, and will render an appropriate widget by default for the following formats:
+##### String formats
+
+The built-in string field also supports the JSONSchema `format` property, and will render an appropriate widget by default for the following string formats:
 
-- `date-time`: Six `select` elements are used to select the year, the month, the day, the hour, the minute and the second;
-    * If you don't want to deal with time, a `"ui:widget": "date"` uiSchema widget is alternatively available, exposing three selects for year, month and day only;
 - `email`: An `input[type=email]` element is used;
 - `uri`: An `input[type=url]` element is used;
-- More formats could be supported in a near future, feel free to help us going faster!
+- `date-time`: By default, an `input[type=datetime-local]` element is used; if you solely want to rely on a date, a `date` uiSchema alternative widget is available:
+
+![](http://i.imgur.com/xqu6Lcp.png)
```

Comment:
There's no easy way to make github serving images hosted in the repo itself. Also, if these are ever gone it's no big deal to redo some captures.